pkgs/tools/X11/xsecurelock/default.nix

```diff
@@ -3,13 +3,14 @@
 , pamtester, xscreensaver }:
 
 stdenv.mkDerivation rec {
-  name = "xsecurelock-git-2018-07-10";
+  name = "xsecurelock-${version}";
+  version = "1.0";
 
   src = fetchFromGitHub {
     owner = "google";
     repo = "xsecurelock";
-    rev = "0fa0d7dd87a4cc4bdb402323f95c3fcacc6f5049";
-    sha256 = "071b3gslszql1mgabs53r82jgbk9mn263m5v6adskfxbkamks8g0";
+    rev = "c11afd6688518eaa657f35e74316c4ed741e8bf9";
```

Comment:
How about `rev = "v${version}";`?

Reply:
Are we happy with tags given that they are mutable? I do prefer the readability of the tag though, and I guess immutability is enforced by the `sha256` too.

Reply:
We prefer to use version tags by default for easier updating and better readability. If we experience problems with moving tags for a particular package we switch it to git revisions later.